src/tests/unit/notification-message.test.tsx

```diff
@@ -26,22 +21,22 @@ const ComponentWithNotificationMessage = ({
   );
 };
 
-describe("useNotification hook", () => {
-  it("Does not show a message if button is not clicked", async () => {
-    const { getByTitle } = render(
-      <ComponentWithNotificationMessage
-        wrapperTitle="test-1"
-        buttonTitle="test-1-button"
-      />
-    );
-    const wrapper = getByTitle("test-1");
+describe("useNotificationMessage hook", () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  it("should not display a message before the button is clicked", async () => {
+    const { getByTestId } = render(<ComponentWithNotificationMessage />);
+    const wrapper = getByTestId("wrapper");
 
+    // Assert that the wrapper does not contain the message initially
     expect(wrapper).toMatchInlineSnapshot(`
       <div
-        title="test-1"
+        data-testid="wrapper"
```

Comment:
It's fine to use data test id's to target the tested elements that is how it is done in other of the similar tests. But don't you get into the problem we saw with everything rendered into the same virtual dom and duplicate id's?

Reply:
I don't think so after I added::
```
 afterEach(() => {
    cleanup();
  });
```